manifests/config.pp

```diff
@@ -61,5 +61,12 @@
     group   => 'root',
     content => $::openvpn::dh,
   }
+  file { '/etc/openvpn/keys/ta.key':
+    ensure  => present,
+    mode    => '0600',
+    owner   => 'root',
+    group   => 'root',
+    content => $::openvpn::ta,
```

Comment:
dh is definitely mandatory according to the ERB template:

https://github.com/astrosoc/puppet-openvpn/blob/master/templates/openvpn.conf.erb#L9

The config will be broken if `dh` is not specified :(